docs/streams.md

```diff
@@ -5,91 +5,83 @@ EventStoreDB is purpose-built for event storage. Unlike traditional state-based
 These **events** are logically organized into **streams**, typically only one stream per entity. 
 
 ## Metadata and reserved names
+#### Stream metadata ####
```

Comment:
I am unfamiliar with this syntax. Is the second set of `####` doing something special or is this still an h4? If it's an h4, I think it should stay as an h3 instead since it's best SEO practice to not jump a header level.

Reply:
Its an h4, I'll change it back to h3 :)